dlrover/python/elastic_agent/torch/training.py

```diff
@@ -1420,17 +1437,19 @@ def run(self, role: str = DEFAULT_ROLE) -> bool:
                 f"Network check time of round {i} is {elapsed_time}"
                 f" and succeed is {result}."
             )
+
+            success = success or result
```

Comment:
When round 1 is a success but round 2 is a failure, it still reports success as the final result. Why?

Reply:
The result reported here is the result status of the current node, not the result status of the group it participates in. The first time being successful and the second time failing indicates that it has been affected by others.